src/main/docker/Dockerfile

```diff
@@ -63,6 +60,8 @@ RUN chgrp 0 /usr/local/bin/docker-entrypoint.sh && \
     chmod g=u /etc/passwd && \
     chmod 0775 /usr/local/bin/docker-entrypoint.sh
 
+RUN rm -rf /usr/share/elasticsearch/jdk
+
```

Comment:
Removing the JDK like this adds an extra layer to the Docker image that removes it's metadata.  To actually save the space it needs to be removed in the builder image so that it isn't present when the `/usr/share/elasticsearch/` directory is initially copied into this image.

Reply:
The plugin install failed if removed earlier, since discovered we had to define JAVA_HOME before installing the plugin.